resources/definitions/flsun_qq.def.json

```diff
@@ -0,0 +1,48 @@
+{
+	"version": 2,
+	"name": "FLSUN QQ",
+	"inherits": "fdmprinter",
+	"metadata": {
+		"visible": true,
+		"manufacturer": "FLSUN",
+		"author": "Daniel Green",
+		"file_formats": "text/x-gcode",
+		"icon": "icon_ultimaker2",
+		"platform_offset": [0, 0, 0],
```

Comment:
This offset is useless if there is no platform mesh set. It's also set to the default value. It doesn't hurt to have it in here, but you might as well remove it.

Reply:
icon is useless as well; it does not exist and it is not used